dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/storage/datastore/DataStoreHandler.kt

```diff
@@ -55,9 +57,11 @@ interface DataStoreHandler {
      * This executes on a worker thread and not on the caller thread.
      *
      * @param key name of the datastore entry
+     * @param callback to indicate whether the operation succeeded or not.
      */
     fun removeValue(
-        key: String
+        key: String,
+        callback: DataStoreWriteCallback
```

Comment:
For when we remove the expired datastore, because this is an async operation and therefore we need to know when it's finished in order to be sure that the initialization of the datastoremanager is complete.